api/v1/openapi.yaml

```
@@ -1,14 +1,14 @@
 info:
-  title: 'Gloo Gateway Portal IDP Connect API'
+  title: 'Gloo Portal IdP Connect API'
   version: 1.0.0
-  description: Before you begin, set up Gloo Gateway Portal with your OpenID Connect (OIDC) provider. Then with this IDP Connect API, you can manage the clients and API Products in your OIDC provider that get associated with your Portal applications. For more information, see the [Gloo Gateway Portal docs](https://docs.solo.io/gloo-portal/latest/).
+  description: Before you begin, set up Gloo Portal with your OpenID Connect (OIDC) provider, and install IdP Connect to integrate it with your Gloo Gateway installation. Then, you can use this IdP Connect API to manage the clients in your OIDC provider that are associated with your Portal apps. For more information, see the [Portal OAuth guide](https://docs.solo.io/gateway/latest/portal/guides/frontend-portal/credential-management/oauth/) in the Gloo Gateway docs.
```

Maybe add the self-service use case?
```suggestion
  description: Before you begin, set up Gloo Portal with your OpenID Connect (OIDC) Identity Provider (IdP), and install IdP Connect to integrate it with your Gloo Gateway installation. Then, you can use this IdP Connect API to manage the clients in your OIDC provider that are associated with your Portal apps. This way, your Portal users can provision their own OAuth credentials to access the Portal apps. For more information, see the [Portal OAuth guide](https://docs.solo.io/gateway/latest/portal/guides/frontend-portal/credential-management/oauth/) in the Gloo Gateway docs.
```